raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -978,9 +978,9 @@ def test_initiator_handle_contract_receive_emptyhash_secret_reveal():
         channelidentifiers_to_channels=setup.channel_map,
         pseudo_random_generator=setup.prng,
     )
-    assert len(iteration.events) == 0
-    # make sure the original lock wasn't moved
-    assert transfer.lock.secrethash in setup.channel.our_state.secrethashes_to_lockedlocks
+    assert len(iteration.events) == 3
```

Comment:
To make the test clear on what it is expecting, it's better to change this to assert for the existence of the specific event you're looking for. You can use: [search_for_item](https://github.com/raiden-network/raiden/blob/develop/raiden/tests/utils/events.py#L67) for this.

Reply:
I'll do this in this PR.

Reply:
I guess I'm looking for EventUnlockSuccess, so added an assert for it.